ede-php-autoload/class-loader/core.el

```diff
@@ -76,6 +76,15 @@ Example: (ede-php-autoload--get-path-relative-to-ns \"My\\Ns\\My\\Class\" \"My\\
     "/")
    (or extension ".php")))
 
+(defun ede-php-autoload--make-completion (name type full-name)
```

Comment:
I think this function can be public and considered to be a helper to create autoloaders. What do you think ?